project1/application.py

```diff
@@ -90,4 +91,21 @@ def authenticate():
 @app.route("/logout")
 def logout():
     session.clear()
-    return render_template("registration.html")
\ No newline at end of file
+    return render_template("registration.html")
+
+# @app.route("/bookpage/<string:isbn_id>")
+# def book_details(isbn_id):
+#     book = db1.session.query(Books).filter(Books.isbn == isbn_id).all()
+#     return render_template("bookpage.html", data=book)
+
+@app.route("/bookpage/<string:isbn>")
+def books(isbn):
+    # ISBN = Books.query.get("isbn")
+    isbn = "1416949658"
+    details =  Books.query.filter_by(isbn = ISBN)
```

Comment:
You will get  only exact results.Try to  work on partial results as well.For example,If I enter 141 for isbn,it should display all the books starting with that series.

Reply:
Mam, we have divided our work into 3 parts. search, bookpage and Review. That displaying of books based on the search is being done by priyanka.  My work is that If I had clicked that particular isbn I should display all the details of that particular book.


![Capture](https://user-images.githubusercontent.com/55695589/80302689-31afc300-87c9-11ea-9ea2-e2bf05736c2a.PNG)

Reply:
ok